src/lwc/rd2PauseForm/rd2PauseForm.html

```diff
@@ -66,15 +66,21 @@
                     <lightning-layout multiple-rows="true">
 
                         <lightning-layout-item padding="horizontal-large" class="slds-p-top_large" size="12">
-                            <template if:true={selectedRowsSummary}>
+                            <div if:true={selectedRowsSummary} style="font-style: italic;">
```

Comment:
I've read that for jest tests to work properly, `if:true` and `if:false` should only be used with the `<template>` tag. So, here it might be better to leave the template as the outer wrapper and add an inner wrapper of div or span with the style tag.

Reply:
Did not know that, thank you, the change has been done.